### PR TITLE
Remove Coveralls

### PR DESCRIPTION
**Pull request by fbertsch**
Coveralls is currently making it impossible to merge PRs, since
they are not having the coveralls status reported. Admins also
cannot override this.

### CODE REVIEW

**Comment by fbertsch** [comment]
See also https://bugzilla.mozilla.org/show_bug.cgi?id=1414076

File: runtests.sh [hunk 1]
```diff
@@ -17,8 +17,3 @@ if [ $# -gt 0 ]; then
 else
     coverage run --source=moztelemetry setup.py test
```
Comment (robhudson) [inline runtests.sh]: Should we not run `coverage` until we have a replacement?

File: Dockerfile [hunk 2]
```diff
@@ -5,7 +5,8 @@ ENV SPARK_VERSION=2.0.2
 # install gcc
 RUN apt-get update --fix-missing && \
     apt-get install -y \
-    g++ libpython-dev libsnappy-dev
+    g++ libpython-dev libsnappy-dev \
+    build-essential libssl-dev libffi-dev
```
Comment (robhudson) [inline Dockerfile]: Why the extra packages?

Comment (fbertsch) [reply Dockerfile]: These are dependencies for `cryptography`, see the second commit.

**Comment by fbertsch** [comment]
@robhudson RFAL

**Pull request merged**